util/weekly_build/03_SetupEnv.sh

```diff
@@ -15,14 +15,15 @@ for compiler in $COMPILERS; do
     [[ ${compiler} == *"@="* ]] && envname=build-$template-${compiler/@=/-} || envname=build-$template-${compiler/@/-}
     envdir=$RUNDIR/$RUNID/envs/$envname
     echo "Setting up environment $envname in $envdir"
-    rm -rf $envdir
-    spack stack create env --name $envname --template $template --site $PLATFORM --compiler $compiler
+    if [ ! -d $envdir ]; then
+      spack stack create env --name $envname --template $template --site $PLATFORM --compiler $compiler
+    fi
     cd $envdir
     spack env activate .
     spack config add "config:install_tree:padded_length:${PADDED_LENGTH:-200}"
     # Check for duplicates and fail before doing the "real" concretization:
     spack_wrapper log.concretize concretize --fresh
-    ${SPACK_STACK_DIR:?}/util/show_duplicate_packages.py log.concretize -d -i crtm-fix -i crtm -i esmf
+    ${SPACK_STACK_DIR:?}/util/show_duplicate_packages.py log.concretize -d -i crtm-fix -i crtm -i esmf -i mapl
```

Comment:
This is already in release/1.9.0. Maybe we should merge the release branch back to develop first?

Reply:
Either way is fine with me